Source/SwiftLintFrameworkTests/TestHelpers.swift

```diff
@@ -33,9 +33,18 @@ extension XCTestCase {
 
         if commentDoesntViolate {
             let commentedViolations = ruleDescription.triggeringExamples.flatMap {
-                violations("/** " + $0, ruleDescription)
+                violations("/**\n  " + $0 + "\n */", ruleDescription)
             }
             XCTAssertEqual(commentedViolations, [])
+        } else {
+            let commentedViolations = ruleDescription.triggeringExamples.flatMap {
```

Comment:
you could reuse the `commentedViolations` and `XCTAssertEqual(commentedViolations, Array(count: commentDoesntViolate  ? 0 : ruleDescription.triggeringExamples.count, repeatedValue: ruleDescription)`.